doc/issues.md

```diff
@@ -23,7 +23,7 @@ Returns an array of issues.
 $issues = $client->api('issue')->find('KnpLabs', 'php-github-api', 'closed', 'bug');
 ```
 
-Returns an array of closed issues matching the "bug" term.
+Returns an array of closed issues matching the "bug" term. For more complex searches, use the [search api](/doc/search.md) which supports the advanced github search syntax.
```

Comment:
please use relative links. It will be more reliable if the way to diplay the doc changes